content/en/docs/topics/charts_hooks.md

```diff
@@ -81,7 +81,9 @@ the Job is run.
 For all other kinds, as soon as Kubernetes marks the resource as loaded (added
 or updated), the resource is considered "Ready". When many resources are
 declared in a hook, the resources are executed serially. If they have hook
-weights (see below), they are executed in weighted order. Otherwise, ordering is
+weights (see below), they are executed in weighted order. 
+Starting from Helm 3.1.0 hook resources with same weight are installed in the same 
```

Comment:
Is this accurate? It seems this work was implemented in https://github.com/helm/helm/pull/7499 which is in the 3.2.0 milestone.

Reply:
Assuming the milestone markings are correct, https://github.com/helm/helm/pull/7499 was made available in Helm 3.2.0, so this would be inaccurate.

Reply:
I was looking at the wrong milestone marking of helm/helm#7432. So I verified when the commit was actually released, see https://github.com/helm/helm/compare/v3.1.0...v3.2.0. @bacongobbler is right, its actually 3.20. PR was updated. Thanks for this remark.